packages/core/typings/src/types/core/plugins/config/strapi-server/controllers.ts

```diff
@@ -0,0 +1,12 @@
+import type { Context } from 'koa';
+import { Strapi } from '../../../../..';
+
+export type ControllerReturnType = {
```

Comment:
We shouldn't be using our own type here, we should be using the same type used by createCoreController https://github.com/strapi/strapi/blob/361b318f91416ef5984112ac5627eaff7988d750/packages/core/strapi/src/factories.ts#L22-L27